src/interfaces/earnings.ts

```diff
@@ -27,9 +27,17 @@ import {
   EarningsDayData,
   EarningsUserData
 } from "../types/custom/earnings";
+import { toBN } from "../utils";
 
 const BigZero = new BigNumber(0);
 
+interface StrategiesResponse {
```

Comment:
Moved this up here as I think interfaces/types should live either at the top of the file or in a separate file

Reply:
Makes sense, I think I inlined it because it was only used in that function, but yes this is better to be more consistent